ports/servoshell/desktop/app.rs

```diff
@@ -198,10 +204,12 @@ impl App {
             CompositeTarget::Window
         };
         let mut servo = Servo::new(
+            self.opts.clone(),
+            self.preferences.clone(),
```

Comment:
The embedder's copy of the preferences will be out of date if they're updated at runtime, right? For example, via webdriver. I don't know that we need to solve that, just making sure that my mental model is correct.